src/util/api/index.js

```diff
@@ -177,6 +178,7 @@ export const getNFTClassesPartial = ({
   if (reverse) qsPayload['pagination.reverse'] = reverse;
   if (limit) qsPayload['pagination.limit'] = limit;
   if (key) qsPayload['pagination.key'] = key;
+  if (nocache) qsPayload.nocache = `${new Date().getTime()}`;
```

Comment:
```suggestion
  if (nocache) qsPayload.ts = `${Math.round(new Date().getTime() / 1000)}`;
```

just call the ts ts seems simpler, also maybe round to second?